will set a always missconfigured part to a default setup 
remove the config part (RemovePath, RequantTyp)
from osd plugin setup 

12 Mar 2009 ,Joerg Bornkessel <hd_brummy@gentoo.org>

diff -Naur burn-0.1.0-pre21.orig/setup.c burn-0.1.0-pre21/setup.c
--- burn-0.1.0-pre21.orig/setup.c	2009-03-12 03:33:46.387393024 +0100
+++ burn-0.1.0-pre21/setup.c	2009-03-12 03:37:56.204037636 +0100
@@ -83,11 +83,11 @@
 	//!--- plugin_setup -----------------------------------------------------------
 
 	plugin_setup::plugin_setup():
-			PROCTOOLS_INIT_PROPERTY( RemovePath,          false ),
+			PROCTOOLS_INIT_PROPERTY( RemovePath,          true ),
 			PROCTOOLS_INIT_PROPERTY( CustomDiskSize,      200 ),
 			PROCTOOLS_INIT_PROPERTY( BurnSpeed,           0 ),
 			PROCTOOLS_INIT_PROPERTY( DemuxType,           demuxtype_vdrsync ),
-			PROCTOOLS_INIT_PROPERTY( RequantType,         requanttype_metakine ),
+			PROCTOOLS_INIT_PROPERTY( RequantType,         requanttype_transcode ),
 			PROCTOOLS_INIT_PROPERTY( PreserveLogFiles,    false ),
 			PROCTOOLS_INIT_PROPERTY( DefaultLanguage,     0 ),
 			PROCTOOLS_INIT_PROPERTY( MainMenuStatus,      true ),
@@ -115,7 +115,7 @@
 			PROCTOOLS_INIT_PROPERTY( SkinIndex,           0 ),
 			PROCTOOLS_INIT_PROPERTY( ChaptersMode,        chaptersmode_10 ),
 			PROCTOOLS_INIT_PROPERTY( StoreMode,           storemode_burn ),
-			PROCTOOLS_INIT_PROPERTY( DmhArchiveMode,      true ),
+			PROCTOOLS_INIT_PROPERTY( DmhArchiveMode,      false ),
 			PROCTOOLS_INIT_PROPERTY( DiskSize,            disksize_singlelayer ),
 			PROCTOOLS_INIT_PROPERTY( CutOnDemux,          false ),
 			PROCTOOLS_INIT_PROPERTY( SkipTitlemenu,       true ),
@@ -199,9 +199,9 @@
 		m_defaults = job_defaults();
 
 		Add( new menu::text_item( tr("--- Common settings --------------------------------------------------") ) );
-		Add( new menu::bool_edit_item( tr("Remove path component"), m_setup.RemovePath ) );
+//		Add( new menu::bool_edit_item( tr("Remove path component"), m_setup.RemovePath ) );
 		Add( new menu::list_edit_item( tr("Demux using"), m_setup.DemuxType, demuxtype_strings ) );
-		Add( new menu::list_edit_item( tr("Requant using"), m_setup.RequantType, requanttype_strings ) );
+//		Add( new menu::list_edit_item( tr("Requant using"), m_setup.RequantType, requanttype_strings ) );
 		Add( new menu::number_edit_item( tr("Burn speed"), m_setup.BurnSpeed, 0, 32, tr("unlimited") ) );
 		Add( new menu::bool_edit_item( tr("Preserve logfiles"), m_setup.PreserveLogFiles ) );
 		Add( new menu::list_edit_item( tr("Spare language code"), m_setup.DefaultLanguage, track_info::get_language_codes(), false ) );
